### PR TITLE
EOS Fix Scale on GiANT products

### PR DESCRIPTION
**Pull request by shitong01**
Fix the scale  from cm to mm and flip colorbar

1.) The color bar has been flipped. 
    - Blue is now subsidence (+ve. increase dist wrt Sat LOS)
    -  Red is uplift  (-ve, decrease dist wrt Sat LOS). However, the sign of the values shown remain the same.
2.) Corrected time-series displacement unit in graph. (cm --> mm) Thanks to Cheryl’s investigation.

Same PR as:
https://github.com/earthobservatory/displacement-ts-server/pull/1
image

EOS Displacement TS server:
https://13.250.171.138/ts/?id=displacement-time-series-sbas_S1-TN69-20170813T095721Z-20190622T095804Z-8d993-v0.1/LS-PARAMS.h5

### CODE REVIEW

**Comment by shitong01** [comment]
@pymonger @torresal

File: tomcat/content/thredds/threddsConfig.xml [hunk 1]
```diff
@@ -202,6 +202,7 @@
   <WMS>
     <allow>true</allow>
     <allowRemote>false</allowRemote>
+    <paletteLocationDir>palettes</paletteLocationDir>
```
Comment (shitong01) [inline tomcat/content/thredds/threddsConfig.xml:205]: Got this methodology to change customize palettes from: https://www.unidata.ucar.edu/software/tds/v4.5/reference/ThreddsConfigXMLFile.html#WMS

File: tomcat/content/thredds/palettes/revrainbow.pal [hunk 2]
```diff
@@ -0,0 +1,27 @@
+102 0 0  #dark red
```
Comment (shitong01) [inline tomcat/content/thredds/palettes/revrainbow.pal:1]: Got this be reversing colors here:
https://github.com/Unidata/tds/blob/master/tds/src/main/webapp/WEB-INF/palettes/alg2.pal

File: apps/leaflet_ts/leaflet_ts/static/js/index.js [hunk 3]
```diff
@@ -59,12 +59,12 @@ var WMS_URL="//"+location.hostname+"/thredds/wms/ts/"+getParameterByName("id");
...
     {
-        "name":"RAW Displacement Time Series (cm)",
+        "name":"RAW Displacement Time Series wrt Satellite LOS (mm)",
         "id":"rawts",
         "colorscalerange":"-100,100"
     },
     {
-        "name":"Filtered Displacement Time Series (cm)",
+        "name":"Filtered Displacement Time Series wrt Satellite LOS (mm)",
         "id":"recons",
```
Comment (shitong01) [inline apps/leaflet_ts/leaflet_ts/static/js/index.js:68]: TODO: change it to 
`Filtered Displacement Time Series in the Satellite LOS (mm)`

**Comment by pymonger** [comment]
Great updates and fixes @shitong01. @torresal, please review and merge.

**Review by pymonger — APPROVED** [review]

**Pull request merged**